.github/workflows/worker-delete.yml

```diff
@@ -0,0 +1,44 @@
+name: Delete Deployment
+
+on:
+  delete:
+
+jobs:
+  delete:
+    runs-on: ubuntu-latest
+    name: Delete Deployment
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.10.0"
+
+      - name: Enable corepack
+        run: corepack enable
+
+      - uses: actions/checkout@v4
+
+      - name: Get Deleted Branch Name
+        id: get_branch
+        run: |
+          branch_name=$(echo '${{ github.event.ref }}' | sed 's#refs/heads/##' | sed 's#[^a-zA-Z0-9]#-#g')
+          echo "branch_name=$branch_name" >> $GITHUB_ENV
+      - name: Retrieve and Construct Full Worker Name
+        id: construct_worker_name
+        run: |
+          base_name=$(grep '^name = ' wrangler.toml | sed 's/^name = "\(.*\)"$/\1/')
```

Comment:
won't this grep match multiple lines for example [a sample config](https://developers.cloudflare.com/workers/wrangler/configuration/#sample-wranglertoml-configuration) that has two lines that start with `name = `